docs/vereisten/motiveringsbeginsel.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Een besluit berust op een deugdelijke motivering
+toelichting: Een besluit dat tot stand is gekomen door of met behulp van een algoritme of AI-systeem, dient te berusten op een deugdelijke motivering. 
+status_vereiste:
+- geldend
+levenscyclus:
+- ontwerp
+- dataverkenning-en-datapreparatie
+- ontwikkelen
+- verificatie-en-validatie
+- implementatie
+- monitoring-en-beheer
+bouwblok:
+- governance
```

Comment:
hoort dit niet ook bij transparantie? Heeft ook te maken met uitlegbaarheid?

Reply:
Het raakt elkaar allemaal. Dit is een algemeen beginsel van behoorlijk bestuur die gekoppeld is aan 'besluitvorming' en dat in het besluit moet worden gemotiveerd hoe het algoritme/AI-systeem daar invloed op heeft gehad. Het m.i. dus een specifiek vereiste die ziet op het genomen besluit. Daarmee ben je meteen ook transparant (ervaar het ook als een kip/ei verhaal).